bin/toolbox

```diff
@@ -371,6 +371,16 @@ main() {
         # can modify the user's name, groups, etc, within the container.
         VOLUMES="--volume /tmp:/tmp:rslave"
         test -d "${HOME}" && VOLUMES="$VOLUMES --volume ${HOME}:${HOME}"
+
+        #Add any symlinked directories to the mounts. Right now descends 3 levels
+        #Would be better to make a general purpose search, but having problems excluding things
+        #from python environments, flatpaks, and podman storage
+        readarray -d '' HOME_MOUNTS < <(find ${HOME} -maxdepth 3 -type l -print0)
+        for HOME_MOUNT in "${HOME_MOUNTS[@]}"
+        do
+            test -x $HOME_MOUNT && VOLUMES="$VOLUMES --volume $HOME_MOUNT:$HOME_MOUNT"
+        done
+
```

Comment:
Please make this an extra PR with an description, why this is needed and which problem this solves.
We mount ${HOME} already in the container, why mount everything below ${HOME} a second time into the container?

Reply:
> Please make this an extra PR with an description, why this is needed and which problem this solves. We mount ${HOME} already in the container, why mount everything below ${HOME} a second time into the container?

Yep, I was about to ask the same: what's the use case for this / what's the problem this is solving? It seems it has to do with symlinks, but I'm not sure I fully understand the purpose of it.

Reply:
I'll see what I can do to put this in a second request. I had this problem and came across one other person with this problem on reddit: When you have symlinks in your home directory, they don't get properly mounted in the container. For example, I have my Music folder on an HDD symlinked to my home directory and the rest of my home directory on an SSD. I think there may be a solution with podman's bind options, but I went through the options one by one, and none of them solved the issue, so I was stuck with this solution.